packages/lib/src/vis/utils.ts

```diff
@@ -350,19 +350,39 @@ export function toArray(arr: NumArray): number[] {
   return isTypedArray(arr) ? [...arr] : arr;
 }
 
-export function worldToCamera(camera: Camera, worldPt: Vector3) {
-  return worldPt.clone().project(camera);
+export function worldToHtml(
+  camera: Camera,
+  context: VisCanvasContextValue,
+  worldPt: Vector3
+) {
+  const cameraPt = worldPt.clone().project(camera);
+  return context.cameraToHtml(cameraPt);
+}
+
+export function htmlToWorld(
+  camera: Camera,
+  context: VisCanvasContextValue,
+  htmlPt: Vector2
+) {
+  return context.htmlToCamera(htmlPt).unproject(camera);
```

Comment:
I don't know if you did it but it would be nice to check that `worldToHtml(htmlToWorld(vec))` gives the same vector (more or less due to numerical precision)